src/components/Home/Home.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './Home.scss';
+import AppAdvert from '../AppAdvert/AppAdvert';
+import MenuLeft from '../MenuLeft/MenuLeft';
+
+function Home() {
+  return (
```

Comment:
You can create an arrow function for this: 
```suggestion
const Home = () => (
```